subsys/net/buf.c

```diff
@@ -78,8 +74,8 @@ static inline struct net_buf *pool_get_uninit(struct net_buf_pool *pool,
 
 void net_buf_reset(struct net_buf *buf)
 {
-	NET_BUF_ASSERT(buf->flags == 0U);
-	NET_BUF_ASSERT(buf->frags == NULL);
+	__ASSERT_NO_MSG(buf->flags == 0U);
+	__ASSERT_NO_MSG(buf->frags == NULL);
```

Comment:
I'd vote for mapping `NET_BUF_ASSERT` onto `__ASSERT_NO_MSG`, perhaps that is less politically contentious and still permits netbuf-specifics if so needed.

Reply:
I didn't seem like it was politically contentious. All voted for removing, and I got no reply to my follow up question. So seems removing completely was fine.

Reply:
Alright :+1: